recipes/multiping/meta.yaml

```diff
@@ -47,35 +47,30 @@ test:
   # Some packages might need a `test/commands` key to check CLI.
   # List all the packages/modules that `run_test.py` imports.
   imports:
-    - simplejson
-    - simplejson.tests
+    - multiping
 
 about:
-  home: https://github.com/simplejson/simplejson
+  home: https://github.com/romana/multi-ping
   # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
   # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
   # See https://opensource.org/licenses/alphabetical
-  license: MIT
+  license: APACHE
   # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
-  license_family: MIT
+  license_family: APACHE
   # It is strongly encouraged to include a license file in the package,
   # (even if the license doesn't require it) using the license_file entry.
   # See https://conda.pydata.org/docs/building/meta-yaml.html#license-file
-  license_file: LICENSE.txt
-  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  license_file: LICENSE
```

Comment:
Please make a pr to submit the license file to the sdist upstream.

Reply:
Thanks for the feedback.
Happy to do it... if I better understood what to do here (sorry, I'm new to this).  I copied this license file directly from the package repo, so I am not sure that I follow where it needs to be submitted.

Reply:
Done, [here](https://github.com/romana/multi-ping/pull/29).